ethexe/rpc/src/lib.rs

```diff
@@ -201,4 +203,9 @@ impl FusedStream for RpcReceiver {
 }
 
 #[derive(Debug)]
-pub enum RpcEvent {}
+pub enum RpcEvent {
+    OffchainTransaction {
+        transaction: SignedOffchainTransaction,
+        response_sender: oneshot::Sender<Result<H256>>,
```

Comment:
I'd put here Option with custom Clone impl in favor of ease of use for @playX18 in his PR of  global events broadcasting

Reply:
done